script.js

```diff
@@ -1,4 +1,88 @@
+var rollDice = function () {
+  var numberRolled = Math.floor(Math.random() * 6) + 1;
+  return numberRolled;
+};
+var player1Scores = [];
+var player2Scores = [];
+var player1Dice1 = 0;
+var player1Dice2 = 0;
+var player2Dice1 = 0;
+var player2Dice2 = 0;
+var player1Total = 0;
+var player2Total = 0;
```

Comment:
declaring too many global variables can cause your code to look very cluttered! try using arrays instead to store information that is similar. you can then access this information using the array's indexes